source/stdafx.h

```diff
@@ -22,7 +22,7 @@
 #include <game_sa/CPools.h>
 #include <game_sa/CMenuManager.h>
 #include <game_sa/CText.h>
-#include <game_sa/CRGBA.h>
+#include <game/CRGBA.h>
```

Comment:
Curious about this. What is the context of this change?

Reply:
This file was moved https://github.com/DK22Pac/plugin-sdk/commit/ff4017fc6afe421b7984b7b389cc88837a2a787e